forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt

```diff
@@ -0,0 +1,16 @@
+Name:Saruman of Many Colors
+ManaCost:3 W U B
+Types:Legendary Creature Avatar Wizard
+PT:5/4
+K:Ward:Discard<1/Enchantment;Instant;Sorcery>
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigMill | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, each opponent mills two cards. When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
+SVar:TrigMill:DB$ Mill | Defined$ Opponent | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBImmediateTrigger
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | RememberObjects$ Remembered | CopyTriggeringObjects$ True | Execute$ TrigExile | SubAbility$ DBCleanup | TriggerDescription$ When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
```

Comment:
why did you add RememberObjects again?

Reply:
Was trying stuff since it's currently not working (anymore)

Reply:
ok CopyTriggeringObjects$ doesn't perform good enough, I wanted to get rid of it eventually anyway

So just Remember the TriggeredCard instead